pss/pss_test.go

```diff
@@ -55,8 +54,7 @@ var (
 	initOnce        = sync.Once{}
 	loglevel        = flag.Int("loglevel", 2, "logging verbosity")
 	longrunning     = flag.Bool("longrunning", false, "do run long-running tests")
-	w               *whisper.Whisper
-	wapi            *whisper.PublicWhisperAPI
+	cryptoBackend   CryptoBackend
```

Comment:
just `crypto`

Reply:
Same as above. `crypto` will conflict with `github.com/ethereum/go-ethereum/crypto` unless we encapsulate all its uses into CryptoBackend (or rename the import as ethCrypto)

Reply:
Changed in commit 49a3b06